Add documentation for introspection parameter on graphql and graphql_sync (#47)
mirumee/ariadne-website: docs/api-reference.md

## `graphql`

```python

> async def graphql(schema, data, *, root_value=None, context_value=None, logger=None, debug=False, introspection=True, validation_rules, error_formatter, extensions=None, middleware, **kwargs)

Reviewer: ```suggestion
async def graphql(
    schema,
    data,
    *,
    root_value=None,
    context_value=None,
    logger=None,
    debug=False,
    introspection=True,
    validation_rules,
    error_formatter,
    extensions=None,
    middleware,
    **kwargs
)
```

If `True` will cause the server to include debug information in error responses.

> #### `introspection`

Reviewer: ```suggestion

#### `introspection`
```

> If `False` will prevent clients from introspecting the schema, returning an error in the response.

Reviewer: ```suggestion
If `False` will prevent clients from introspecting the schema, returning an error when any of introspection fields such as `__schema` are queried.

Disabling introspection will also disable the GraphQL Playground, resulting in error 405 "method not allowed" being returned for GET requests.
```

## `graphql_sync`

```python

> def graphql(schema, data, *, root_value=None, context_value=None, debug=False, introspection=True, validation_rules, error_formatter, middleware, **kwargs)

Reviewer: ```suggestion
def graphql(
    schema,
    data,
    *,
    root_value=None,
    context_value=None,
    debug=False,
    introspection=True,
    validation_rules,
    error_formatter,
    middleware,
    **kwargs
)
```
